docs/app/pages/components/components-sections/media-player/media-player/media-player.component.html

```diff
@@ -46,6 +52,9 @@
     <tr uxd-api-property name="source" type="string">
         The url to the media file to be loaded by the media player.
     </tr>
+    <tr uxd-api-property name="displayName" type="string">
```

Comment:
@donna-marijne Would this be better as `filename`? Or do you think `displayName` describes what this is better?

I think the description could also be re-worded to follow the phrasing we use in our other component inputs. It might also be worth mentioning that this only applies to audio files? And that by default the file name is retrieved from the source file?

Reply:
Yes, either `filename` or `title` would be fine.

Reply:
Great, I think it is best to avoid `title` as if someone uses this as a string attribute this will also add a browser tooltip to the element.

Reply:
Good point!